pkgs/development/python-modules/lmfit/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, buildPythonPackage, fetchPypi,
+  setuptools-scm, asteval, numpy, scipy, uncertainties,
+  pytest-cov, pytestCheckHook, pandas, matplotlib,
+}:
+
+buildPythonPackage rec {
+  pname = "lmfit";
+  version = "1.2.1";
+
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-AM71vRRb+BtzYwt4kmrySyTxgFQh5iEcpYVYiqfMQVs=";
+  };
+
+  propagatedBuildInputs = [
+    setuptools-scm
+    asteval
+    numpy
+    scipy
+    uncertainties
+  ];
+
+  nativeCheckInputs = [
+    pytest-cov
```

Comment:
We generally don't run coverage related tests. Remove this package and substitute the --cov argument here: https://github.com/lmfit/lmfit-py/blob/master/setup.cfg#L99